includes/admin/class-wcmp-export.php

```diff
@@ -334,9 +337,18 @@ public function add_shipments(array $order_ids, bool $process)
 
             $extraOptions = WCX_Order::get_meta($order, WCMYPA_Admin::META_SHIPMENT_OPTIONS_EXTRA);
             $colloAmount  = $extraOptions["collo_amount"] ?? 1;
+            $isMailboxPackage  = 'mailbox' === $this->getPackageTypeFromOrder($order);
+            $isMultiColloAllowed = (
+                self::COUNTRY_CODE_NL === $order->get_shipping_country() ||
+                self::COUNTRY_CODE_BE === $order->get_shipping_country()
+            );
 
-            if ($colloAmount > 1) {
+            if ($colloAmount > 1 && $isMultiColloAllowed && ! $isMailboxPackage) {
                 $collection->addMultiCollo($consignment, $colloAmount);
+            } else if ($isMailboxPackage && $colloAmount > 1) {
```

Comment:
Je hebt hier een aantal verwarrende dubbele condities, iets als dit is al leesbaarder:

```php
if ($colloAmount > 1 && $isMultiColloCountry) {
    if ($isMailboxPackage) {
        // Mailbox packages don't support real multicollo.
        for ($i = 1; $i <= $colloAmount; $i++) {
            $collection->addConsignment($consignment);
        }
    } else {
        $collection->addMultiCollo($consignment, $colloAmount);
    }
} else {
    $collection->addConsignment($consignment);
}
```

Reply:
Ik heb hier nu het een en ander veranderd. Alleen hoeft er voor de mailbox package geen multicollo country te zijn